requirements.txt

```diff
@@ -1,8 +1,9 @@
-Django==1.11.29                  # core server-side framework. Behind a major version due to missing Python 2 support
+Django==3.0.8                    # core server-side framework. Behind a major version due to missing Python 2 support
```

Comment:
I was actually planning to bump a lot of these packages in the coming weeks anyways! Good to know that it doesn't break seqr to do so :)

Reply:
The old version won't work with the `django-allauth` package.